Code/GraphMol/FileParsers/testAtropisomers.cpp

```diff
@@ -567,6 +569,32 @@ void testLookForAtropisomersInSDdfFiles(std::string fileName,
   TEST_ASSERT(notFoundCount == expectedMisses);
 }
 
+void testSulfinamideExamplesHaveNoAtropisomers() {
+  const std::vector<std::string> controlFiles = {
+      "sulfinamide-double-bond-O-R.mol",
+      "sulfinamide-single-bond-O-R.mol",
+      "sulfinamide-single-bond-O-S.mol",
+  };
+  std::string rdbase = getenv("RDBASE");
+  std::stringstream warningCapture;
+  rdWarningLog->SetTee(warningCapture);
+  for (const auto &file : controlFiles) {
+    auto fName = rdbase +
+                 "/Code/GraphMol/FileParsers/test_data/atropisomers/" +
+                 file;
+    BOOST_LOG(rdInfoLog) << "Validating absence of atropisomers in " << file
+                         << std::endl;
+    auto mol = std::unique_ptr<RWMol>(MolFileToMol(fName, true, false, false));
+    TEST_ASSERT(mol);
+    const Conformer *conf =
```

Comment:
The molecule must have a conformer at this point.
You don't need to use the ternary operator here.